app/models/phone.rb

```diff
@@ -0,0 +1,19 @@
+class Phone < ApplicationRecord
+  belongs_to :contact
+
+  validates :number, presence: true, uniqueness: {
+    scope: :contacts,
```

Comment:
Esta sendo possivel criar varios telefones com o mesmo numero.

```suggestion
    scope: :contact,
```